app/views/protocols/view_details/_research_involving.html.haml

```diff
@@ -27,7 +27,7 @@
     %h5
       %strong
         = t(:protocols)[:studies][:research_involving][:header]
-        
+
   - if research_types_info.present?
```

Comment:
@wtholt It looks like there will not always be a Research Types info based on my current data. I don't think I was very clear though. I meant we can combine the two into `if research_types_info.present? && (research_types_info.human_subjects || research_types_info.vertebrate_animals || ... )`. That way even if research_types_info is nil, it won't check against the children types